packages/hash/backend-utils/src/logger.ts

```diff
@@ -22,9 +37,23 @@ export class Logger {
 
   constructor(private cfg: LoggerConfig) {
     this.cfg = cfg;
+
     const logger = winston.createLogger({
-      level: cfg.level,
-      format: winston.format.combine(winston.format.errors({ stack: true })),
+      level: cfg.level ?? getDefaultLoggerLevel(),
```

Comment:
Out of curiosity, when are we planning to create loggers that use level other than `process.env.LOG_LEVEL`?

Reply:
I'm not sure I'm understanding correctly, all logging levels are supported by `process.env.LOG_LEVEL`? (i.e. it can be set to `debug`, `info`, `warn`, or `error`).

If you are referring to supporting package-specific logging levels (i.e. only set `realtime` logs to `debug` level), that's what I had imagined for [this follow-up task](https://app.asana.com/0/1201095311341924/1201829153674958/f). Although you could currently accomplish this by supplying a custom `level` argument to the constructor where the `logger` instance is initialised in one of the packages.

Reply:
Right. So we have `cfg.level` instead of just `getDefaultLoggerLevel()` to vary log level per package. IC, makes sense.